benchmarks/lib/src/jmh/java/com/amazon/corretto/crypto/provider/benchmarks/SignatureMlDSA.java

```diff
@@ -0,0 +1,34 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.corretto.crypto.provider.benchmarks;
+
+import com.amazon.corretto.crypto.provider.AmazonCorrettoCryptoProvider;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+
+@State(Scope.Benchmark)
+public class SignatureMlDSA extends SignatureBase {
+    @Param({AmazonCorrettoCryptoProvider.PROVIDER_NAME, "BC"})
+    public String provider;
+
+    @Param({"ML-DSA-44", "ML-DSA-65", "ML-DSA-87"})
+    public String algo;
```

Comment:
It's important for us to be more consistent with the parameter names because they're used directly as CloudWatch metric dimensions. Can you expand this to `algorithm` to align with other benchmarks?

Reply:
will do.